vulnerable_people_form/integrations/notifications.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import json
+
+import boto3
+from botocore.config import Config
+from flask import current_app
+
+from vulnerable_people_form.form_pages.shared.session import form_answers
+from vulnerable_people_form.form_pages.shared.logger_utils import init_logger
+
+logger = logging.getLogger(__name__)
+init_logger(logger)
+
+boto3_config = Config(
+    retries={
+        "max_attempts": 5,
+        "mode": "standard",
+    }
+)
+
+
+def get_client_kwargs(app=current_app):
+    return {
+        "endpoint_url": app.config.get("LOCAL_SQS_ENDPOINT_URL"),
+        "aws_access_key_id": app.config.get("AWS_ACCESS_KEY"),
+        "aws_secret_access_key": app.config.get("AWS_SECRET_ACCESS_KEY"),
+        "region_name": app.config.get("AWS_REGION"),
+    }
+
+
+def get_sqs_client(app=current_app):
+    return boto3.client("sqs", **get_client_kwargs(app))
+
+
+def send_message(registration_number, app=current_app):
+    client = get_sqs_client()
+
+    postcode = form_answers()["support_address"]["postcode"]
+    if postcode[-4] != " ":
+        postcode = f"{postcode[:-3]} {postcode[-3:]}"
```

Comment:
https://github.com/alphagov/govuk-shielded-vulnerable-people-service/pull/226 adds a new function `postcode_with_spaces` which would be good to substitute here if it goes in first

Reply:
Updated to use this function